ext/RMagick/extconf.rb

```diff
@@ -323,9 +281,7 @@ def assert_minimum_ruby_version!
     def assert_has_dev_libs!
       return unless RUBY_PLATFORM !~ /mswin|mingw/
 
-      # check for pkg-config if Magick-config doesn't exist
-      if $magick_config && `Magick-config --libs`[/\bl\s*(MagickCore|Magick)6?\b/]
-      elsif $pkg_config && `pkg-config --libs MagickCore`[/\bl\s*(MagickCore|Magick)6?\b/]
+      if `pkg-config --libs MagickCore`[/\bl\s*(MagickCore|Magick)6?\b/]
```

Comment:
Can we change this to an `unless` and remove the `else`?

Reply:
Oh, good catch!!! I will modify that. Thank you!!